### ISSUE TITLE
Android notification icon

### ISSUE THREAD

**Issue by xyboox** [phonegap/phonegap-plugin-push#2167]
### Expected Behaviour
Icon should be placed in /platforms/android/res directory ( as per docs ), but there is no such directory anymore in my Ionic 3 project, I guess since Cordova v8 ( please see the screenshot )
![screenshot at jan 26 10-42-28](https://user-images.githubusercontent.com/642584/35431791-9bb07a08-0286-11e8-87e8-f7222c135805.png)

### Actual Behaviour
-

### Reproduce Scenario (including but not limited to)
start an empty project and add android platform

#### Steps to Reproduce 
-

#### Platform and Version (eg. Android 5.0 or iOS 9.2.1)
all android

#### (Android) What device vendor (e.g. Samsung, HTC, Sony...)
-

#### Cordova CLI version and cordova platform version

    cli packages: (/usr/local/lib/node_modules)

    @ionic/cli-utils  : 1.19.1
    ionic (Ionic CLI) : 3.19.1

global packages:

    cordova (Cordova CLI) : 8.0.0 

local packages:

    @ionic/app-scripts : 3.1.6
    Cordova Platforms  : android 7.0.0 ios 4.5.4
    Ionic Framework    : ionic-angular 3.9.2

System:

    Android SDK Tools : 26.1.1
    ios-deploy        : 1.9.2 
    Node              : v8.9.4
    npm               : 5.6.0 
    OS                : macOS High Sierra
    Xcode             : Xcode 9.2 Build version 9C40b 

#### Plugin version

    cordova plugin version | grep phonegap-plugin-push   # e.g. 1.5.3

#### Sample Push Data Payload
not relevant

#### Sample Code that illustrates the problem
not relevant

#### Logs taken while reproducing problem
not relevant

Thank you!

**Comment by macdonst** [comment]
@xyboox yeah the new folder is `platforms/android/app/src/main/res`, I'll look into updating the docs.

**Comment by xyboox** [comment]
Thanks @macdonst !
